apps/yarn.lock

```diff
@@ -1585,7 +1585,8 @@
   integrity sha512-Aex5zpSmWDvTBRGNJWwI12qYWhXG+u0bnb74LD917EdE94tsz4P2p7B/tby3NZdsE4DvNHjUG0dxCwxTbcitHg==
 
 "@cdo/interpreted@link:../dashboard/config/libraries":
-  version "0.1.0"
+  version "0.0.0"
```

Comment:
is this change related to the dance party version bump?

Reply:
Great question.  I'm not sure where this came from, and it didn't repro when trying a second time, so I'll remove the change.